pkg/client/leaderelection/leaderelection.go

```diff
@@ -169,7 +168,6 @@ type LeaderElectionRecord struct {
 // Run starts the leader election loop
 func (le *LeaderElector) Run() {
 	defer func() {
```

Comment:
https://godoc.org/k8s.io/kubernetes/pkg/util/runtime 

```
TODO: remove this function. We are switching to a world where it's safe for apiserver to panic, since it will be restarted by kubelet. At the beginning of the Kubernetes project, nothing was going to restart apiserver and so catching panics was important. But it's actually much simpler for montoring software if we just exit when an unexpected panic happens.
```